families/battleship/tests/test_tp_battleship.yaml

```diff
@@ -44,16 +53,18 @@ services:
     image: battleship-tests:$ISOLATION_ID
     volumes:
       - $SAWTOOTH_CORE:/project/sawtooth-core
+    command: "bash -c \"
+              nose2-3 \
+              -vvvv \
+              -c /project/sawtooth-core/families/battleship/nose2.cfg \
+              -s /project/sawtooth-core/families/battleship/tests \
+              test_tp_battleship
+              \""
     expose:
       - 4004
-    command: nose2-3
-        -c /project/sawtooth-core/families/battleship/nose2.cfg
-        -v
-        -s /project/sawtooth-core/families/battleship/tests
-        test_tp_battleship
     stop_signal: SIGKILL
     environment:
-        TEST_BIND: "tcp://eth0:4004"
+        TEST_BIND: "tcp://0.0.0.0:4004"
```

Comment:
Do we know why this change is necessary? It is generally bad form to bind to all interfaces.

Reply:
@boydjohnson: Not sure why I had that there, removed that change